modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+
+[id="ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod_{context}"]
+= Using a SharedSecret resource in a pod
```

Comment:
@gabemontero  Does this topic need updates in response to [this comment thread](https://docs.google.com/document/d/1F3VFkiLUJbsUkFJcjnA2oyROYTI5RRlTJYl4ZWQoNls/edit?disco=AAAARrKwSiw) in the gdoc?

Reply:
Will do.